bot.py

```diff
@@ -122,5 +135,13 @@ async def on_message(self, msg: discord.Message):
             "Invalid config. Did you forget to add the bot token to the `.secrets.toml` file? See the README for more info."
         )
         sys.exit(1)
-    bot = MyBot(intents=intents)
+    bot = MyBot(
+        intents=intents,
+        command_prefix=prefix,
+        case_insensitive=True,
+        # dpy comes with a help command, we need to remove it
+        help_command=None,
+        # disallows the bot from mentioning things it shouldn't, just in case
+        allowed_mentions=discord.AllowedMentions(everyone=False, users=True, roles=False, replied_user=True)
```

Comment:
For anyone new to dpy, these are not ulimit style, i/e, any time we want to override this for whatever reason we can, but having it off by default is safe